app/_data/tables/install_options_34x.yml

```diff
@@ -1,5 +1,5 @@
 columns:
-  - name: Platform
+  - name: Runtime
```

Comment:
```suggestion
  - name: Platform
```
This is referring to the column, so the "Docker", "Kubernetes", and "Linux" entries. I don't think runtime is the right term for that?

Reply:
I read it across rather than down. Maybe just empty the cell if it could be confusing

Reply:
Can we relocate the heading for that column so it's clearer where it applies? otherwise yes I agree empty is better

Reply:
Removing for now to see how it looks

Reply:
![Screenshot 2024-04-12 at 2 28 26 PM](https://github.com/Kong/docs.konghq.com/assets/59702069/888c2760-a56e-438b-8c3c-81af3d04ca29)
![Screenshot 2024-04-12 at 2 47 51 PM](https://github.com/Kong/docs.konghq.com/assets/59702069/1da1c407-a3f3-4f65-9f87-ba5fbf8a520d)

Empty is clearer than "Runtime" or "Platform", IMO. @lena-larionova WDYT?

Reply:
I like it, let's go with empty

Reply:
Thanks for the good catch!